cord19-explorer/src/main/webapp/WEB-INF/views/affiliation.html

```diff
@@ -3,29 +3,34 @@
     <title>CORD19 Geo Explorer</title>
   </head>
   <body>
-   <div layout:fragment="content">
+  <div th:replace="layouts/navbar :: navbar"></div>
+   <div layout:fragment="content" class="content">
   
    <h3>[[${institution}]]</h3>
-   
-   	<div class="row">
-   		<div th:each="pub : ${publications}">
-   		  <h4><a th:href="@{'/paper/' + ${pub.paperId}}">[[${#strings.defaultString(pub.metadata.title, 'No title available')}]]</a></h4>
-   		  <ul>
-   		  <li th:each="author : ${pub.metadata.authors}">
-   		  <i>[[${author.affiliation.institution}]], [[${author.affiliation.locationSettlement}]], [[${author.affiliation.locationRegion}]], [[${author.affiliation.locationCountry}]]</i>
-   		  <br>
-   		  <b>Selected affiliation: </b>[[${author.affiliation?.selectedWikiarticle?.title}]] - [[${author.affiliation?.selectedWikiarticle?.coordinates}]]
-   		  <ul>
-   		  	<li th:each="wiki : ${author.affiliation?.wikiarticles}">
-   		  	[[${wiki.title}]]
-   		  	</li>
-   		  </ul>
-   		  </li>
-   		  </ul>
-		    
-	    </div>
-   	</div>
-     
-   </div>
+
+		<div th:each="pub : ${publications}">
+			<h4>
+				<a th:href="@{'/paper/' + ${pub.paperId}}">[[${#strings.defaultString(pub.metadata.title,
+					'No title available')}]]</a>
+			</h4>
+			<ul style="color:white;">
+				<li th:each="author : ${pub.metadata.authors}"><i>[[${author.affiliation.institution}]]<span th:if="${author.affiliation.locationSettlement}">,</span>
+						[[${author.affiliation.locationSettlement}]]<span th:if="${author.affiliation.locationRegion}">,</span>
+						[[${author.affiliation.locationRegion}]]<span th:if="${author.affiliation.locationCountry}">,</span>
+						[[${author.affiliation.locationCountry}]]</i> <br> <b>Selected
+						affiliation: </b><a
+					th:href="@{https://en.wikipedia.org/wiki/__${author.affiliation?.selectedWikiarticle?.title}__}"
+					target=_blank>[[${author.affiliation?.selectedWikiarticle?.title}]]</a>
+					<span th:if="${clean != null}">- [[${author.affiliation?.selectedWikiarticle?.coordinates}]]</span>
+					<div th:if="${clean != null}">
+					<ul>
+						<li th:each="wiki : ${author.affiliation?.wikiarticles}">
+							[[${wiki.title}]]</li>
+					</ul></li>
+			</ul>
+			</div>
+		</div>
+
+	</div>
```

Comment:
just create an extra page; likelihood is that we want more changes down the road and that the internal and external views diverge even further.

Reply:
and remove the cleaning parameter